composer.json

```diff
@@ -1,9 +1,14 @@
 {
   "name": "friendsoftypo3/widgets",
   "type": "typo3-cms-extension",
-  "description": "Dashboard Widgets for TYPO3",
+  "description": "Widgets Collection",
```

Comment:
I'd suggest to keep "Dashboard"... e.g. "Dashboard Widget Collection" or "Widget Collection for ext:dashboard"

Reply:
The idea behind this change was to remove redundancy. In TYPO3 context widgets only appear on the dashboard - as far as i am aware of - so as long as this set, the keyword "dashboard" only adds redundancy here. See e.g. the "Matomo Widgets" project (https://extensions.typo3.org/extension/matomo_widgets/) which concluded the same.

Reply:
The extension you referenced says
> **Dashboard** widgets with Matomo reports

The term "widget" has used in combination with Fluid already and "widget" might exist in other areas as well as this term is very generic. Having an additional reference to "dashboard" just helps users to grasp the scope easily.

Reply:
Yes, they use "dashboard" in their abstract as we do in the first sentence of the abstract, but not in the title.

But i did not encounter Fluid widgets, so it might be good to separate from them. Will add "dashboard" though.

Reply:
Done.